js/ui/cmp/About.js

```diff
@@ -37,7 +37,7 @@ ui.cmp.About = Ext.extend(Ext.Window,
                             '</div>' +
                             '<div id="phd-oe-about-info">' + PhDOE.app.name + //' ver ' + PhDOE.app.ver + '<br/>' +
                                 //'UI: ' + PhDOE.app.uiRevision + '<br/>' +
-                                ' Copyright &copy; 2008-2012 The PHP Group<br/>' +
+                                ' Copyright &copy; 2008-' + new Date().format('Y') + ' The PHP Group<br/>' +
```

Comment:
Suggest [`new Date().getFullyear()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear).

Reply:
@salathe thanks, fixed it.
The previous format was similar to [this code](https://github.com/php/web-doc-editor/blob/master/js/ui/cmp/PortletInfo.js#L288)

Reply:
Ah, I forgot that ExtJS adds that method.  Feel free to use whichever you choose. 😄

Reply:
Hmm ... for a uniform code style, the format will probably be better?
If you agree with me - I'll send a new commit :)

Reply:
Sure, that sounds fine.

Reply:
done :)